domains/anuj.json

```
@@ -0,0 +1,12 @@
+
+{
+  "description": "Anuj' Welcome to my portfolio",
+  "repo": "https://anujmutha.github.io/Portfolio/",
+  "owner": {
+    "username": "AnujMutha",
+    "email": "mutha.am16@gmail.com"
+  },
+  "record": {
+    "CNAME": "anujmutha.github.io/Portfolio/"
```

Please remove the `/Portfolio/` part of the record.
```suggestion
    "CNAME": "anujmutha.github.io"
```